### DISCUSSION TITLE
Watts API for getting power consumption data

### DISCUSSION THREAD

**Discussion by Parker-McDonnell** [tnmendes/watt#134] [Ideas]
I don't know much about web development,but I was wondering how difficult it would be for you to expose the data collected by this app through something like an API token (I don't know what I'm talking about 😂) so I could read data from the app to a website that I build?

**Comment by tnmendes** [comment]
Hi @Parker-McDonnell 

At this time there is no way to expose the content that Watt gets to outside the app. The most that Watt allows is to expose it for use in Apple's Shortcuts app.

However, there are several projects on Github that you can use to obtain device consumption data and then make the graphical interface.
